### PR TITLE
IsStillRunning logging under debug

### PR DESCRIPTION
**Pull request by pveentjer**
No more warning logging, but everything under debug.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/IsStillRunningService.java [hunk 1]
```diff
@@ -86,7 +87,9 @@ public boolean isOperationExecuting(Invocation invocation) {
         } catch (Exception e) {
             invocation.logger.warning("While asking 'is-executing': " + invocation, e);
         }
-        invocation.logger.warning("'is-executing': " + executing + " -> " + invocation);
+        if (invocation.logger.isFineEnabled()) {
+            invocation.logger.finest("'is-executing': " + executing + " -> " + invocation);
```
Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/IsStillRunningService.java]: You check for `isFineEnabled()` but use `finest()`. There is a `isFinestEnabled()` method which should most probably be used here.

File: hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/IsStillRunningService.java [hunk 2]
```diff
@@ -224,7 +229,9 @@ public IsOperationStillRunningCallback(Invocation invocation) {
         @Override
         public void onResponse(Object response) {
             boolean executing = Boolean.TRUE.equals(response);
-            invocation.logger.warning("'is-executing': " + executing + " -> " + invocation);
+            if (invocation.logger.isFineEnabled()) {
```
Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/IsStillRunningService.java]: Same here as above.

File: hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/IsStillRunningService.java [hunk 3]
```diff
@@ -264,7 +271,9 @@ public void run() {
                     invocation.nodeEngine, invocation.serviceName, isStillRunningOperation,
                     invocation.getTarget(), 0, 0, IS_EXECUTING_CALL_TIMEOUT, callback, true);
 
-            invocation.logger.warning("Asking if operation execution has been started: " + invocation);
+            if (invocation.logger.isFineEnabled()) {
```
Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/IsStillRunningService.java]: And a third time.

**Comment by jerrinot** [comment]
moving to 3.6.1 as discussed

**Comment by jerrinot** [comment]
:+1:

**Comment by pveentjer** [comment]
A fix has been made that prevent running into this problem in the first place. So we can close this PR

**Pull request closed without merging**